apps/site/assets/ts/stop/components/DepartureCard.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { ReactElement } from "react";
+import { routeBgClass, busClass } from "../../helpers/css";
+import renderFa from "../../helpers/render-fa";
+import { breakTextAtSlash } from "../../helpers/text";
+import { isASilverLineRoute } from "../../models/route";
+import { Route } from "../../__v3api";
+import CRsvg from "../../../static/images/icon-commuter-rail-default.svg";
+import Bussvg from "../../../static/images/icon-bus-default.svg";
+import SubwaySvg from "../../../static/images/icon-subway-default.svg";
+import FerrySvg from "../../../static/images/icon-ferry-default.svg";
+import renderSvg from "../../helpers/render-svg";
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const TypeToModeIcon = (route: Route): any => {
+  switch (route.type) {
+    case 0:
+    case 1:
+      return SubwaySvg;
+
+    case 2:
+      return CRsvg;
+
+    case 3:
+      return Bussvg;
+
+    case 4:
+      return FerrySvg;
+
+    default:
+      return null;
+      break;
+  }
+};
+
+const DepartureCard = ({
+  route
+}: {
+  route: Route;
+}): ReactElement<HTMLElement> => {
+  const RouteName = (
+    <span className={busClass(route)}>
+      {isASilverLineRoute(route.id)
+        ? `Silver Line ${route.name}`
+        : breakTextAtSlash(route.name)}
+    </span>
+  );
+  return (
+    <li className="departure-card">
+      <div className={`departure-card__route ${routeBgClass(route)}`}>
+        {renderSvg("c-svg__icon", TypeToModeIcon(route), true)} {RouteName}
+      </div>
+      {Object.entries(route.direction_destinations).map(
```

Comment:
This will render in whatever order the object happens to be in, which might not be stable over time. Is there a deliberate sort we should do here?